src/OnlineSales/DTOs/CommentDtos.cs

```diff
@@ -40,14 +40,26 @@ public class CommentDetailsDto : CommentCreateDto
 
 public class CommentImportDto : CommentCreateDto
 {
+    private DateTime? createdAt;
+
+    private DateTime? updatedAt;
+
     [Optional]
     public int? Id { get; set; }
 
     [Optional]
-    public DateTime? CreatedAt { get; set; }
+    public DateTime? CreatedAt

```

Comment:
Please remove that complexity from Dtos, they should not carry any extra business logic. Why not to use some kind of global flag in DBContext which you can set during the import?